umbral/utils.py

```diff
@@ -20,35 +21,37 @@ def lambda_coeff(id_i, selected_ids):
 
     return result
 
+
 def poly_eval(coeff, x):
     result = coeff[-1]
     for i in range(-2, -len(coeff) - 1, -1):
         result = ((result * x) + coeff[i])
 
     return result
 
+
 # minVal = (1 << 256) % self.order   (i.e., 2^256 % order)
 MINVAL_SECP256K1_HASH_256 = 432420386565659656852420866394968145599
 
-def hash_to_bn(list, params):
+
+def hash_to_bn(crypto_items, params):
     digest = hashes.Hash(hashes.SHA256(), backend=default_backend())
-    for x in list:
-        if isinstance(x, Point):
-            bytes = x.to_bytes()
-        elif isinstance(x, BigNum):
-            bytes = int(x).to_bytes(32, byteorder='big')
+    for item in crypto_items:
+        if isinstance(item, Point):
+            data_bytes = item.to_bytes()
+        elif isinstance(item, BigNum):
+            data_bytes = int(item).to_bytes(32, byteorder='big')
         else:
-            # print(type(x))
-            bytes = x
-        digest.update(bytes)
+            data_bytes = item
+        digest.update(data_bytes)
 
     i = 0
     h = 0
     while h < MINVAL_SECP256K1_HASH_256:
         digest_i = digest.copy()
         digest_i.update(i.to_bytes(32, byteorder='big'))
-        hash = digest_i.finalize()
-        h = int.from_bytes(hash, byteorder='big', signed=False)
+        hash_digest = digest_i.finalize()
```

Comment:
Next thing you know, @tuxxy is going to show a sudden interest in cache invalidation.  ;-)